crates/cards/tests/basic.rs

```diff
@@ -0,0 +1,88 @@
+#![allow(dead_code, clippy::unwrap_used, clippy::panic_in_result_fn)]
```

Comment:
```suggestion
#![allow(clippy::unwrap_used)]
```

I don't think we violate the other 2 lints here.